extensions/ql-vscode/src/extension.ts

```diff
@@ -1558,6 +1559,10 @@ async function activateWithInstalledDistribution(
   const dbModule = await initializeDbModule(app);
   ctx.subscriptions.push(dbModule);
 
+  const dbSelectionDecorationProvider = new DbSelectionDecorationProvider();
```

Comment:
Can we move this to the DbModule? Would be great to keep everything related to DB initialisation in one place.